Sources/ArcGISToolkit/Components/Augmented Reality/WorldScaleSceneView.swift

```diff
@@ -15,10 +15,14 @@
 import ARKit
 import SwiftUI
 import ArcGIS
+#if os(visionOS)
+import CoreLocation
+#endif
```

Comment:
As far as I can tell, this framework is available on visionOS. Why does this import statement need to be conditionally compiled?

Reply:
Oh this one was interesting. If I remove these few lines of code here and compile for visionOS I will get this error:

> Cannot find 'CLLocationManager' in scope

but only for visionOS not the other platforms. So that is why I added this import.

Reply:
I get adding the import. My question is why not import it on other platforms? I suggest doing that as the code in the file is clearly dependent on it on all platforms. It must be implicitly imported (probably by ARKit), but there's nothing wrong with making the dependency explicit.